app/questionnaire/placeholder_renderer.py

```diff
@@ -108,7 +116,9 @@ def render(
         pointers = find_pointers_containing(dict_to_render, "placeholders")
 
         for pointer in pointers:
-            rendered_text = self.render_pointer(dict_to_render, pointer, list_item_id)
+            rendered_text = self.render_pointer(
```

Comment:
Worth adding and enforcing named attributes in `render_pointer` as there is four of them now, but not a blocker.

Reply:
Enforcing this one as well now 👍